cpp/foxglove/tests/test_server.cpp

```diff
@@ -11,3 +12,16 @@ TEST_CASE("Start and stop server") {
   REQUIRE(server.port() != 0);
   server.stop();
 }
+
+TEST_CASE("Log a message") {
+  foxglove::WebSocketServerOptions options;
+  options.name = "unit-test";
+  options.host = "127.0.0.1";
+  options.port = 0;
+  foxglove::WebSocketServer server{options};
+  REQUIRE(server.port() != 0);
+
+  foxglove::Channel channel{"example", "json", std::nullopt};
+  const uint8_t data[] = {1, 2, 3};
```

Comment:
Does this not work?
```suggestion
  const std::byte data[] = {1, 2, 3};
```

Reply:
nope, integer literals can't be assigned without a cast, so I could do `std::byte(1), ...`...